spec/support/pages/accounts_edit_page.rb

```diff
@@ -31,22 +33,40 @@ def save
   end
 
   def add_category(account_type, category_name)
-    add_category_input = if account_type == :asset
-                           find('.m-accountType:nth-child(1) input[placeholder="add category"]')
-                         else
-                           find('.m-accountType:nth-child(2) input[placeholder="add category"]')
-                         end
-    add_category_input.set(category_name)
-    add_category_input.trigger('blur')
+    find(
+      ".m-accountType:nth-child(#{account_type == :asset ? 1 : 2})" \
+      ' input[placeholder="add category"]'
+    ).instance_eval do
+      set category_name
+      trigger 'blur'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.